content/download.html

```diff
@@ -26,6 +26,7 @@ <h2>What do I do with this?</h2>
 
 <h2>Additional downloads</h2>
 <p><a href="http://downloads.sourceforge.net/reactos/ReactOS-{{< reactos-download-version >}}-src.zip">Source code for ReactOS {{< reactos-version >}}</a></p>
+<p><a href="https://fosstorrents.com/distributions/reactos/">ReactOS Torrent downloads via FossTorrents</a></p>
```

Comment:
I have checked via Google, they write their org name as "FOSS Torrents"
```suggestion
<p><a href="https://fosstorrents.com/distributions/reactos/">ReactOS Torrent downloads via FOSS Torrents</a></p>
```